### PR TITLE
[#52549] 13.3 Documentation: Create and share custom project lists wi…

### PR DESCRIPTION
**Pull request by MayaBerd**
…th new filter options  https://community.openproject.org/work_packages/52549

[#52549] 13.3 Documentation: Create and share custom project lists with new filter options

https://community.openproject.org/work_packages/52549

### CODE REVIEW

**Comment by MayaBerd** [comment]
@as-op I moved some pages and headings around here, could you please check if this broke any links? I am afraid it might.

**Review by as-op — APPROVED** [review]

**Comment by as-op** [comment]
Links are adjusted.

File: docs/user-guide/home/global-modules/README.md [hunk 1]
```diff
@@ -30,17 +30,9 @@ You can create a new project here directly by using the  **+ Project** button.
 
 ![OpenProject projects overview in the global modules menu](openproject_global_modules_projects.png)
 
-On the left you will have the following options:
-
 **All projects** is the default view and shows only active projects. If you do not have administrator rights, this view will be the same as **My projects**.
```
Comment (marcalcobe) [inline docs/user-guide/home/global-modules/README.md]: The first entry in the menu is no longer called "All projects" but "Active projects"

File: docs/user-guide/projects/projects-lists/README.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+  title: Projects lists
+  priority: 800
+description: Manage projects lists in OpenProject
+keywords: projects lists, project filters, project overview
+
+---
+
+# Projects lists
+
+In OpenProject you can create projects to collaborate with your team members, track issues, document and share information with stakeholders, organize things. A project is a way to structure and organize your work in OpenProject.
```
Comment (marcalcobe) [inline docs/user-guide/projects/projects-lists/README.md]: I think the last coma after stakeholders should be an "and"

File: docs/user-guide/projects/projects-lists/README.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+There are several ways to get an overview of all your projects. You can press the **Projects lists** button at the bottom of the **Select a project** menu in the top left header navigation. 
+
+![projects list button](Projects-list-button.png)
+
+Alternatively, you can use the [**Global modules menu**](../../home/global-modules/#projects) and select *Projects*. You can access that menu either on the left side of the OpenProject application homepage or by using the grid menu icon in the top left corner.
+
+![Select all projects from the global modules menu in OpenProject](view_all_projects_options.png)
+
+You will then get a list of all your projects in OpenProject.
```
Comment (marcalcobe) [inline docs/user-guide/projects/projects-lists/README.md]: I would include "of all active projects in OpenProject"

File: docs/user-guide/projects/projects-lists/README.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+You will then get a list of all your projects in OpenProject. 
+
+![A list of all projects in OpenProject](projects-list.png)
+
+### Projects lists view explained
+
+You can use this projects overview to **create a multi project status dashboard** if you include your own [project custom fields](../../system-admin-guide/custom-fields/custom-fields-projects/), e.g. custom status options, Accountable, Project duration, and more.
+
+> **Please note:** Project custom fields are an Enterprise add-on and will only be displayed here for Enterprise on-premises and Enterprise cloud.
```
Comment (marcalcobe) [inline docs/user-guide/projects/projects-lists/README.md:42]: Don't know if that is maintained as Enterprise or not at the end @oliverguenther

File: docs/user-guide/projects/projects-lists/README.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+Alternatively you can also use the **Activity module** from the [global modules menu](../../home/global-modules/#activity).
+
+![Select activity from the global modules menu](activity-global-menu.png)
+
+By selecting *Overall activity* you can open a view in which all the latest global project activities are documented. In the menu on the left side you can filter the activity by different areas to control e.g. the activity of work packages, wiki pages or meetings.
+
+![overall activity meeting filter](actvity-global-filter.png)
+
+
+## Export projects
```
Comment (marcalcobe) [inline docs/user-guide/projects/projects-lists/README.md]: I think here should be export project lists and not export projects

**Review by marcalcobe — CHANGES_REQUESTED** [review]
Hi @MayaBerd, I've added some small comments on the review.

File: docs/user-guide/projects/projects-lists/README.md (hunk 4 above)
Comment (MayaBerd) [reply docs/user-guide/projects/projects-lists/README.md:42]: @oliverguenther please let me know and I can adjust it throughout documentation later on. thanks!

**Pull request merged**